### ISSUE TITLE
Eliminate line breaks in recipient address

### ISSUE THREAD

**Issue by morchickit** [ThreeSixtyGiving/standard#261]
"Eliminate line breaks in recipient address and other fields; street address should be on one line and should not contain the city and country of the recipient.
Feedback from the Foundation Center. Anyway, Can CoVE check street address fields only contains street and not city and country?

**Comment by robredpath** [comment]
> Anyway, Can CoVE check street address fields only contains street and not city and country?

I don't think we can do this directly, but we could use a proxy. For example, we could make an additional check that flagged multiple commas and/or line breaks in the address field. We could try to come up with a rule that looked for known city or county names at appropriate places, but it's very hard to do that in a way that doesn't create tons of false positives or is so inaccurate as to be useless - to a computer, what's the difference between "51, London Road" and "41 Nottingham Road, London Borough of Islington"? 

Just a format check might get us a good amount of the way through, though. I've added https://github.com/OpenDataServices/cove/issues/1176
